pkg/skaffold/build/buildpacks/logger.go

```diff
@@ -0,0 +1,61 @@
+/*
+Copyright 2019 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package buildpacks
+
+import (
+	"io"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/buildpacks/pack/logging"
+)
+
+type logger struct {
```

Comment:
Would be worth a comment here to say this logger exists to meet the requirements of the pack logger.